torch/_dynamo/output_graph.py

```diff
@@ -585,6 +585,12 @@ def compile_subgraph(
                     output.append(create_instruction("POP_TOP"))
             self.add_output_instructions(output + pass2.get_instructions())
 
+        from .variables.tensor import NumpyTensorVariable
+
+        if stack_values and isinstance(stack_values[-1], NumpyTensorVariable):
+            self.add_output_instructions(
+                stack_values[-1].reconstruct_ndarray(stack_values, tx)
+            )
```

Comment:
So what's the deal? Why do we still have to do this?

Reply:
Yeah trying to figure this out. Let me remove this thing and add `reconstruct` support in next PR.